resource/subfunctions/resource_class.md

```diff
@@ -0,0 +1,323 @@
+---
+layout: default
+grand_parent: General Resource Admin
+parent: Subfunctions
+---
+
+# Resource Class
+
+Resource Administration functions for Resource Class Administration
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note }
+> _This may not be the only step to manage a **Resource Class** in your environment._
+> _You may also have to refresh the **CDT** class to enact these changes._
+> _Please consult RACF documentation and manuals for an understanding of the **CDT** class._
+
+## `ResourceAdmin.add_resource_class()`
+
+```python
+  def add_resource_class( self, class_name: str, traits: dict = {} ) -> Union[dict, bytes]:
+```

```

Comment:
```python
def delete_kerberos_realm(self, kerberos_realm_name: str) -> Union[dict, bytes]:
```

Reply:
Remove leading spaces.

Reply:
Leading spaces and unnecessary trailing spaces and indents removed across all function definitions.